packages/suite-desktop-core/e2e/support/fixtures.ts

```diff
@@ -91,8 +104,12 @@ const test = base.extend<Fixtures>({
         const walletPage = new WalletActions(window);
         await use(walletPage);
     },
-    onboardingPage: async ({ window }, use) => {
-        const onboardingPage = new OnboardingActions(window);
+    onboardingPage: async ({ window, emulatorStartConf }, use, testInfo) => {
+        const onboardingPage = new OnboardingActions(
+            window,
+            emulatorStartConf.model ?? TrezorUserEnvLink.defaultModel,
```

Comment:
Do you really want to make this tied to what is in tenv link? I can see the logic, but it could also create problems.